backend/game_engine/common.py

```diff
@@ -225,17 +225,19 @@ def log_message(message: Text, game_id: Text = None, additional_tags: Dict = Non
         # Sentry automatically pushes exceptions. To avoid this in local env, only init sentry when needed
         init_sentry()
     with push_scope() as scope:
-        if additional_tags:
-            for tag, value in additional_tags.items():
-                logging.info(tag +  '->' + value)
-                scope.set_tag(tag, value)
-
         if game_id:
             logging.info("game_id " + game_id)
```

Comment:
nit (optional): consider using python3 style string formatting in general. i.e.

logging.info("game_id {}".format(game_id))